setup.py

```diff
@@ -45,7 +45,7 @@ def package_data(pkg, roots):
 
 setup(
     name='xblock-poll',
-    version='1.8.9',
+    version='1.9',
```

Comment:
We'll get to the full review next sprint, but for the version I think you should keep the patch number, so 1.9.0

Reply:
@xitij2000, this doesn't look like a big change, so shouldn't we just bump the patch version (1.8.10)?
I think bumping minor would be more suitable for a PR like #66.

Reply:
@Agrendalath  Yeah, I wasn't commenting on the version bump itself, but mainly the format of the version number. I think 1.8.10 makes more sense here.

Reply:
@Agrendalath I have bumped the version as well. Please go ahead

Reply:
cc: @xitij2000 as the reviewer here.